cloudbuild/python-scripts/insert_dynamic_partitions.py

```diff
@@ -0,0 +1,146 @@
+"""
+Python script to dynamically partitions to a BigQuery partitioned table.
+"""
+import argparse
+import random
+from collections.abc import Sequence
+import datetime
+import threading
+import time
+from absl import app
+from utils import utils
+
+
+def wait():
+    print(
+        'Going to sleep, waiting for connector to read existing, Time:'
+        f' {datetime.datetime.now()}'
+    )
+    # This is the time connector takes to read the previous rows
+    time.sleep(2.5 * 60)
+
+
+def main(argv: Sequence[str]) -> None:
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        '--now_timestamp',
+        dest='now_timestamp',
```

Comment:
Not a great name. Something like `execution_timestamp` makes more sense. Btw, why do we need this argument?

Reply:
Fixed the name. We need this  argument to make sure we insert the partitions for the current date. If we provide a fixed date, then they will not be read (as partitions have been marked as closed) .

Reply:
Current date can be found during execution using python's time or date libraries. Why do we need to take that as input?